recipes/configs/phi3/mini_qlora_single_device.yaml

```diff
@@ -40,52 +42,50 @@ checkpointer:
   model_type: PHI3_MINI
 resume_from_checkpoint: False
 
-# Dataset and Sampler
+# Dataset
 dataset:
   _component_: torchtune.datasets.alpaca_cleaned_dataset
 seed: null
 shuffle: True
-batch_size: 2
 
-# Optimizer and Scheduler
+# Fine-tuning arguments
+epochs: 1
+max_steps_per_epoch: null
+batch_size: 2
```

Comment:
bsz + grad acc + default gpus called out in example command above should be 128 if possible. 

source: me, anecdotally. Fight me